spec/jobs/update_topic_hot_scores_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "file_store/s3_store"
+
+RSpec.describe Jobs::UpdateTopicHotScores do
+  let :job do
+    subject
+  end
+
+  fab!(:topic) { Fabricate(:topic, created_at: 1.day.ago) }
+
+  it "runs an update even if hot is missing from top_menu (once every 6 hours)" do
+    SiteSetting.top_menu = "latest"
+    job.class.clear_once_a_day_cache!
```

Comment:
```suggestion
```

I don't think this is required since Redis starts from a clean slate before each test.

Reply:
its tricky isn't it... if anyone forgets about the snapshotting stuff later one ... will remove and remove the clear cache method.

Reply:
Come to think of it, we should maybe just ship redis snapshotting for all examples assuming it doesn't slow down tests too much.